docs/local_tutorial.md

```diff
@@ -0,0 +1,233 @@
+# Solr on Kubernetes on local Mac
```

Comment:
I commented on slack that I think this should be a more generalized example (i.e not specific to mac).

That would perhaps mean using minibike/k3d/KinD instead...

Reply:
Would definitely love to have documentations for all of those options!

Are you using any of those other providers? If so would you be willing to help out with those steps?

Reply:
I have continued to develop the initial tutorial into a lab for a training course. For that I added instructions for Windows (which is a big hassle tbh). Feel free to copy/paste something from here: https://gist.github.com/4bc3398822b8b68ec94c60a05474e409

Since it is not mac-only I do not assume brew or anything, but have included choco commands. Powershell / CMD is a mess, so I added separate examples for them. Can you imagine that Powershell has a default alias for `curl` to some internal cmdled, even if you have the real curl.exe on path. ;(

Reply:
That's great @janhoy!

I really wish there was a way in markdown to select an environment at the top and have that determine which example you saw. That would make the experience a whole lot cleaner.